Pod/Classes/NSAttributedString+BonMotUtilities.m

```diff
@@ -0,0 +1,61 @@
+//
+//  NSAttributedString+BonMotUtilities.m
+//  BonMot
+//
+//  Created by Eliot Williams on 3/9/16.
+//
+//
+
+#import "NSAttributedString+BonMotUtilities.h"
+#import "BONChain.h"
+#import "BONSpecial.h"
+
+@implementation NSAttributedString (BonMotUtilities)
+
+- (NSString *)humanReadableString
+{
+    NSString *originalString = self.string;
+    NSMutableString *composedHumanReadableString = [NSMutableString string];
+
+    [originalString enumerateSubstringsInRange:NSMakeRange(0, originalString.length) options:NSStringEnumerationByComposedCharacterSequences usingBlock:^(NSString *BONCNullable substring, NSRange substringRange, NSRange enclosingRange, BOOL *BONCNonnull stop) {
+
+        static NSCharacterSet *s_newLineCharacterSet = nil;
+        if (!s_newLineCharacterSet) {
+            s_newLineCharacterSet = [NSCharacterSet newlineCharacterSet];
+        }
+        NSCharacterSet *s_whiteSpaceAndNewLinesSet = nil;
+        if (!s_whiteSpaceAndNewLinesSet) {
+            s_whiteSpaceAndNewLinesSet = [NSCharacterSet whitespaceAndNewlineCharacterSet];
+        }
+
+        BONGeneric(NSDictionary, NSNumber *, NSString *)*specialCharacterSubstitutionNameDictionary = [BONSpecial humanReadableStringDictionary];
+        unichar character = [substring characterAtIndex:0];
+        NSString *specialCharacterSubstitutionString = specialCharacterSubstitutionNameDictionary[@(character)];
```

Comment:
#pp this can be made less wordy with:

```objc
NSString *specialCharacterSubstitutionString = [BONSpecial humanReadableStringDictionary][@(character)];
```

Not required, and there are valid reasons to do it your way, but I think that would be my approach.